helm/chart/maesh/crds/smi-spec.yaml

```diff
@@ -5,27 +5,92 @@ metadata:
   name: httproutegroups.specs.smi-spec.io
 spec:
   group: specs.smi-spec.io
-  version: v1alpha1
```

Comment:
To be consistent with the CRD name I think we should rename this file to `smi-specs.yaml`.
wdyt?